Project.toml

```diff
@@ -1,6 +1,6 @@
 name = "CategoricalArrays"
 uuid = "324d7699-5711-5eae-9e2f-1d82baa6b597"
-version = "0.9.0"
+version = "0.10.0"
```

Comment:
Better release 0.9.1 with this change as it's not breaking. We can move to 0.10 later if needed.

Reply:
done